### PR TITLE
Tagging 134

### PR DESCRIPTION
**Pull request by FrankMittelbach**
# Internal housekeeping

## Status of pull request

- Feedback wanted 
- Ready to merge

## Checklist of required changes before merge will be approved
- [x] Test file(s) added
- [x] Version and date string updated in changed source files
- [x] Relevant `\changes` entries in source included
- [x] Relevant `changes.txt` updated
- [n/a] Rollback provided (if necessary)?
- [x] `ltnewsX.tex` (and/or `latexchanges.tex`) updated

### CODE REVIEW

File: base/doc/ltnews40.tex [hunk 1]
```diff
@@ -278,6 +294,25 @@ \section{Improvement to \XeTeX\ \cs{showhyphens}}
...
+In the last release we added tagging support for \pkg{array},
+\pkg{longtable} and other tabular packages, but we overlooked that the
+kernel definition for \cs{cline} also needs modification because the
+rule generated by the command needs to be tagged as an
+artifact. Furthermore, the processing of a \cs{cline} looks to the
+algorithm as if another row is added (which is technically what
+happens), thus it was also necessary to decrement the internal row
+counter to get a correct row count. This has now been corrected in
+\pkg{array} which is automatically loaded for tagging, so that all
+these packages are no fully compatible with the tagging code if it is
```
Comment (davidcarlisle) [inline base/doc/ltnews40.tex:323]: are NOW fully

File: required/tools/array.dtx [hunk 2]
```diff
@@ -3517,7 +3517,46 @@ Bug reports can be opened (category \texttt{#1}) at\\%
...
+  \@multicnt#2
+  \advance\@multicnt-#1
+  \advance\@multispan\@ne
+%    \end{macrocode}
+%    The rule needs artifact tagging in tagged PDF.
+%    \begin{macrocode}
+  \UseSocket{tagsupport/tbl/leaders/begin}
+  \leaders\hrule\@height\arrayrulewidth\hfill
+  \UseSocket{tagsupport/tbl/leaders/end}
+%    \end{macrocode}
```
Comment (u-fischer) [inline required/tools/array.dtx:3549]: Shouldn't that be \UseTaggingSocket{tbl/leaders/begin} etc?

**Review by davidcarlisle — COMMENTED** [review]
one trivial comment typo, I also note Ulrike's comment on the \useTaggingSocket or \UseSocket, other than that look sgood

File: required/tools/array.dtx (hunk 2 above)
Comment (FrankMittelbach) [reply required/tools/array.dtx:3549]: yes it should

**Review by davidcarlisle — APPROVED** [review]

**Review by u-fischer — APPROVED** [review]

**Pull request merged**